tools/install.sh

```diff
@@ -176,6 +176,7 @@ echo "Ubuntu distribution detected."
         echo $yn | tee -a $LOGFILE > /dev/null
         case $yn in
             [Yy]* )
+                export projectname=""
```

Comment:
Why is this exported?

Reply:
I was getting an error without putting that keyword in front. I'm not that familiar with bash in this regard.

Do you know whether `declare` would work better?

Reply:
if you were getting an error without it it's fine, I'll merge it. It just looks out of place.